integration-tests/aws/pom.xml

```diff
@@ -152,7 +152,6 @@
                                     <enableHttpUrlHandler>true</enableHttpUrlHandler>
                                     <enableServer>false</enableServer>
                                     <dumpProxies>false</dumpProxies>
-                                    <enableJni>true</enableJni>
```

Comment:
Hmmm, so, JNI is not enabled by default in 19.2 and we will support both. So if you need JNI, you should add a `JniBuildItem` in your extension.

Reply:
we'll have a look